File: Java/NavigationDraWAct/app/src/main/java/com/example/navigationdrawact/ui/tools/ToolsViewModel.java
package com.example.navigationdrawact.ui.tools;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class ToolsViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public ToolsViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is Despacho fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: xProjectDemo/app/src/main/java/com/example/xprojectdemo/ui/tools/ToolsViewModel.kt
package com.example.xprojectdemo.ui.tools

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel
import com.example.xprojectdemo.ui.home.mApp

class ToolsViewModel : ViewModel() {

    private val _text = MutableLiveData<String>().apply {
        value = "This is tools Fragment"

        mApp.xGlobalsVar.image="popo3"
    }
    val text: LiveData<String> = _text

}